azblob/zc_sas_query_params.go

```diff
@@ -25,15 +25,15 @@ const (
 func FormatTimesForSASSigning(startTime, expiryTime, snapshotTime time.Time) (string, string, string) {
 	ss := ""
 	if !startTime.IsZero() {
-		ss = startTime.Format(SASTimeFormat) // "yyyy-MM-ddTHH:mm:ssZ"
+		ss = startTime.UTC().Format(SASTimeFormat) // "yyyy-MM-ddTHH:mm:ssZ"
```

Comment:
Hey @someone1 ,

Thanks for reaching out !
We are happy to see your contribution.

There are couple of things which I'd like to point out
1. We always merge to `dev` branch before releasing it to master. So I changed the base to `dev` branch
2. There is a conflict in merge which will be removed if you do the required changes in either `formatSASTimeWithDefaultFormat()` function or send UTC time as argument itself.

Also, it'd be nice if you can see whether `formatSASTimeWithDefaultFormat()` func is getting used some other function call because if that is the case, we need to see if UTC time is required in that function or not?